htdocs/view/layout/header.php

```diff
@@ -104,19 +104,32 @@
   </li>
 
   <li>
-		<?php echo link_to(path("logout", "home"), "<i class=\"fa fa-fw fa-power-off\" style=\"color:#fff;\"></i>") ?>
+    <?php echo link_to(path("logout", "home"), "<i class=\"fa fa-fw fa-power-off\" style=\"color:#fff;\"></i>") ?>
   </li>
 
   <!-- Modal : the user can choose the wave to ask for subsidies (make a request) -->
   <?php
     if (isset($_GET["prefix"]) && $_GET["prefix"] == "binet" && has_editing_rights($binet, $term)) {
         ob_start();
-        $waves_for_modal = select_waves(array("state" => "submission"), "submission_date", false);
+        $waves_for_modal = select_waves(array("state" => array("submission", "deliberation")), "submission_date", false);
```

Comment:
mh ... tu as testé ? C'est pas prévu que ça marche ça. La syntaxe correcte c'est : select_waves(array("state" => array("IN", array("submission", "deliberation"))), "submission_date", false);

Mais j'avoue que ton idée est pas mal, je verrai plus tard si je ne peux pas changer. ce qu'il serait possible de faire, c'est : 
select_waves(array("state IN" => array("submission", "deliberation")), "submission_date", false);
tu pense que cette syntaxe est plus lisible ?